README.md

```diff
@@ -1 +1,165 @@
-# CATALOG
\ No newline at end of file
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents 
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [📸 Video presentation](#video-presentation)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Catalog of my things] <a name="about-project"></a>
+
+**[Catalog of my things]** is a console application that allows users to manage collections of things they own. It contains book, music album, movie and game classes. Everything is based on the UML diagram.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Ruby</summary>
+  <ul>
+    <li>Ruby</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Ruby classes]**
+- **[Preserve data]**
+- **[Ruby associations]**
+- **[Unit tests]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+  git clone git@github.com:2004-okari/CATALOG/.git
+
+
+Example command:
+
+```sh
+  bin/rails test test/models/article_test.rb
+
+```
+
+### Install
+
+Install this project with:
+
+### Usage
+
+To run the project, execute the following command:
+
+install Ruby
+
+### Run tests
+
+To run tests, run the following command:
+
+rspec
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Rooney Okari**
+
+- GitHub: [2004-Okari](https://github.com/2004-okari)
+- LinkedIn: [Okari-Nyandika](https://www.linkedin.com/in/rooney-okari-86a5ba250/)
+
+👤 **Bhekisile Jozi**
+
+- GitHub: [@Bhekisile](https://github.com/Bhekisile)
+- LinkedIn: [BhekisileJozi](https://www.linkedin.com/in/bhekisile-j-b9823369/)
+
+👤 **Newton Ombese**
+
+- GitHub: [@newtonOmbese](https://github.com/mohammadnewton)
+- LinkedIn: [@newtonOmbese](https://www.linkedin.com/in/mohammad-newton-ombese/)
+
+👤 **Nweneary Uzochukwu Winnie**
+
+- GitHub: [@wineshuga](https://github.com/wineshuga)
+- LinkedIn: [@wineshuga](https://linkedin.com/in/wineshuga)
+- LinkedIn: [@wineshuga](https://twitter.com/wineshuga) 
+
+<!-- VIDEO PRESENTATION -->
+
+## 🔭 Video presentations <a name="video-presentation"></a>
+
+- [Link](https://www.loom.com/share/c36ddcd15994058b73071bc912aaf9?sid=f1595b5-3028-4f78-8982-a091887e0c8)
```

Comment:
- [ ] @Team great work adding the presentation, but it seems it doesn't allow anybody with the link to view it,I am getting the error below when I click the link 
![image](https://github.com/2004-okari/CATALOG/assets/61408860/169825a0-6b9a-4290-96d0-dad3f45bb4c7)